templates/environment/cf.yml

```diff
@@ -329,6 +329,12 @@ Resources:
               "logs:GetLogDelivery"
             ]
             Resource: "*"
+          - Sid: Cloudwatch
+            Effect: Allow
+            Action: [
+              "cloudwatch:DescribeAlarms"
+            ]
+            Resource: "*"
```

Comment:
Can you check to see if we can include a tag based condition here? We can only describe alarms tagged with ecs-project/env

Reply:
will do!